src/app/components/change-email/change-email.component.html

```diff
@@ -0,0 +1,61 @@
+<mat-drawer-container class="example-container">
```

Comment:
This drawer should be it's own component because it is duplicated in the manage-access component

Reply:
Do I just need to change the name then? Like to "change-email-container" for example?